internal/cmd/service/service.go

```diff
@@ -31,6 +32,7 @@ func NewCmdService(f *cmdutil.Factory) *cobra.Command {
 	cmd.AddCommand(serviceRedeployCmd.NewCmdRedeploy(f))
 	cmd.AddCommand(serviceSuspendCmd.NewCmdSuspend(f))
 	cmd.AddCommand(serviceDeployCmd.NewCmdDeploy(f))
+	cmd.AddCommand(serviceDomainCmd.NewCmdCreateDomain(f))
```

Comment:
The command seems to be registered as a sub command in domain already?

Reply:
> The command seems to be registered as a sub command in domain already?

fixed